__mocks__/POST.mock

```diff
@@ -0,0 +1,17 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+{{#code}}
+(() => {
+    const jwtToken = "mocked_jwt_token";
+
+    return {
+        status: 200,
+        headers: {
+            'X-Requested-By': 'register/user'
+        },
+        body: JSON.stringify({
+            jwt: jwtToken,
+        })
+    };
+})();
+{{/code}}
```

Comment:
please place this mock response file into the right folder. 
ask Tetiana for help if necessary.